pkg/operation/botanist/logging.go

```diff
@@ -198,7 +188,7 @@ func (b *Botanist) DefaultVali() (component.Deployer, error) {
 		v1beta1constants.PriorityClassNameShootControlPlane100,
 		nil,
 		b.ComputeValiHost(),
-		ingressClass,
+		v1beta1constants.SeedNginxIngressClass,
```

Comment:
It is always this ingress class now, right? Hence, we don't need to have it configurable anymore?